.github/workflows/main.yml

```diff
@@ -176,6 +186,22 @@ jobs:
         with:
           name: tests-performance
           path: Snowflake.Data.Tests/linux_${{ matrix.dotnet }}_${{ matrix.cloud_env }}_performance.csv
+      # codecov action sometimes fails but retry works - until action provides retry let's retry on our own https://github.com/codecov/codecov-action/issues/926
```

Comment:
Retries may not work, but @sfc-gh-igarish can setup CodeCov token in secrets and we can then use the token as in https://github.com/snowflakedb/snowflake-connector-nodejs/pull/608

Reply:
Ok. setup the code cove token. Please check.

Reply:
Codecov token is working. Thanks.